File: src/main/java/gargoyle/ct/prop/impl/simple/CTSimpleByteProperty.java
package gargoyle.ct.prop.impl.simple;

import gargoyle.ct.prop.CTNumberProperty;
import org.jetbrains.annotations.NotNull;

public class CTSimpleByteProperty extends CTSimpleProperty<Byte> implements CTNumberProperty<Byte> {
    public CTSimpleByteProperty(String name) {
        this(name, (byte) 0);
    }

    public CTSimpleByteProperty(String name, @NotNull Byte def) {
        super(name, def);
    }
}

File: src/main/kotlin/gargoyle/ct/prop/impl/simple/CTSimpleByteProperty.kt
package gargoyle.ct.prop.impl.simple

import gargoyle.ct.prop.CTNumberProperty

class CTSimpleByteProperty @JvmOverloads constructor(name: String, def: Byte = 0.toByte()) :
    CTSimpleProperty<Byte>(name, def), CTNumberProperty<Byte>
